meta-cube/recipes-core/lxc/files/lxc-overlayrestore

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -
+
+# /usr/lib64/lxc/lxc/lxc-overlayrestore
```

Comment:
For the sake of portability, we shouldn't hard code this path. We should make sure this command is on the path, versus having the absolute directory in the script